tests/lib/snaps/apply-config-hook-set/meta/hooks/apply-config

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+echo "Setting a configuration value"
+if ! snapctl set baz=qux; then
```

Comment:
This isn't actually possible without `snap set`, so I suggest we land as-is and clean up in the follow-up PR making `snap set` work.